README.md

```diff
@@ -86,6 +86,8 @@ Define AAD_TENANT, AAD_CLIENT environment variables - e.g. locally in .env envir
 ```bash
 AAD_TENANT=[tenant (directory) ID]
 AAD_CLIENT= [client (application) ID]
+AAD_API_SCOPE= [API Scopes Required for Downstream APIs, in our case Microsoft Graph API]
+AAD_CLIENT_SECRET=Client secret obtained from the AAD App Registration
```

Comment:
Is this not environment specific i.e. that it is the client registration for the playground environment? Am not too worried about this leaking out as it is only our playground setup.

Reply:
Addressed in the latest commit.